docs/docs/getting-started/first-steps.md

```diff
@@ -170,6 +170,16 @@ If you encounter any problem with the following steps, make sure to use the [lat
 
     :::
 
+    :::note
+
+    **AWS**: Currently, confidential VMs may temporarily fail to boot. If creating or initializing a cluster fails, terminate the cluster and try again later. Alternatively, you can use non-confidential Nitro VMs for evaluation purposes. To do so, generate a new configuration with the `aws-nitro-tpm` attestation variant:
```

Comment:
```suggestion
    **AWS**: Currently, confidential VMs may temporarily fail to boot. AWS is actively working on a solution. If creating or initializing a cluster fails, terminate the cluster and try again later. Alternatively, you can use non-confidential Nitro VMs for evaluation purposes. To do so, generate a new configuration with the `aws-nitro-tpm` attestation variant:
```
Mention that this is out of our control? Could also be sth like `We are actively working with AWS to remediate the issue.`